pkg/sac/query_helper.go

```diff
@@ -14,6 +14,16 @@ var (
 // BuildClusterLevelSACQueryFilter builds a Scoped Access Control query filter that can be
 // injected in search queries for resource types that have direct cluster scope level.
 func BuildClusterLevelSACQueryFilter(root *effectiveaccessscope.ScopeTree) (*v1.Query, error) {
+	return buildClusterLevelSACQueryFilter(root, true)
+}
+
+// BuildSilentClusterLevelSACQueryFilter builds a Scoped Access Control query filter that can be
+// injected in search queries for resource types that have direct cluster scope level.
+func BuildSilentClusterLevelSACQueryFilter(root *effectiveaccessscope.ScopeTree) (*v1.Query, error) {
```

Comment:
nit: `BuildNonVerbose..` instead of `Silent`? I was wondering in the beginning what the name actually stands for.